manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -10,6 +10,7 @@ auditChunkSize: 0
 logLevel: INFO
 logDenies: false
 emitAdmissionEvents: false
+experimentalEnableMutation: false
```

Comment:
Even though mutations is still in alpha right now, I wonder if this should be 
```
mutations:
  enabled: false
```
or something similar. Once mutations is released we would have to rename this variable, which doesn't seem ideal.